README.md

```diff
@@ -88,6 +88,16 @@ workers:
   transfer: 10
 ```
 
+Excluding directories or files to be scraped from the source can be reached with specifying a regex in the job definition:
+```yaml
+jobs:
+  - from: http://de.archive.ubuntu.com/ubuntu/
+    to:   mirror/ubuntu-repos
+    excl: /sub_dir$|.gz$
+```
+
+This would exclude directories named `sub_dir` and files with extension `gz` on every level.
```

Comment:
Ah, nevermind. I was reading the wrong branch. Still, I would find it worthwhile to append a `/` to the string being tested against the regex if it's a directory, since it allows to distinguish directories and files in the regex.